src/up.cmd.js

```diff
@@ -307,6 +314,25 @@ class UpCommand extends BuildCommand {
 
         await this._project.start();
 
+        // init source watcher after everything is built
```

Comment:
wait until the project is started before the watcher is initialized. this used to cause some problems when the helix-pages checkout was copied to `.hlx/pages/master`.